lib/semantics/check-do-stmt.h

```diff
@@ -12,30 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef FORTRAN_SEMANTICS_CHECK_DO_CONCURRENT_H_
-#define FORTRAN_SEMANTICS_CHECK_DO_CONCURRENT_H_
+#ifndef FORTRAN_SEMANTICS_CHECK_DO_STMT_H_
+#define FORTRAN_SEMANTICS_CHECK_DO_STMT_H_
 
 #include "semantics.h"
-#include "../common/indirection.h"
 
 namespace Fortran::parser {
 struct DoConstruct;
 }
-namespace Fortran::semantics {
-class DoConcurrentContext;
-}
-extern template class Fortran::common::Indirection<
-    Fortran::semantics::DoConcurrentContext>;
 
 namespace Fortran::semantics {
-class DoConcurrentChecker : public virtual BaseChecker {
+
+class DoStmtChecker : public virtual BaseChecker {
 public:
-  explicit DoConcurrentChecker(SemanticsContext &);
```

Comment:
The definition for this and other DoConcurrentChecker class members also need to be removed or renamed right?

They exist in chec-do-concurrent.cc file.

Reply:
You can disregard this comment. The diff of check-do-concurrent.cc file was not showing before, I had to manually load the diff.